qkernel/src/kernel_def.rs

```diff
@@ -401,18 +401,17 @@ pub fn child_clone(userSp: u64) {
     CPULocal::SetKernelStack(currTask.GetKernelSp());
 
     currTask.AccountTaskEnter(SchedState::RunningApp);
-    let pt = currTask.GetPtRegs();
 
-    let kernelRsp = pt as *const _ as u64;
+    let pt_regs_top = currTask.GetPtRegs() as *const _ as u64;
     CPULocal::Myself().SetEnterAppTimestamp(TSC.Rdtsc());
     //currTask.mm.VcpuEnter();
     CPULocal::Myself().SetMode(VcpuMode::User);
     currTask.mm.HandleTlbShootdown();
-    debug!("entering child task: kernelSp/PtRegs @ {:x}", kernelRsp);
+    debug!("entering child task: kernelSp => PtRegs @ {:x}", pt_regs_top);
```

Comment:
why the arrow?

Reply:
Stack pointer point to the begging of the PtRegs.